[DOCS] restructure the docs (#993)
elastic/apm-pipeline-library: docs/CONTRIBUTING.md

# Guidance on how to contribute

**Table of contents:**

1. [Using the issue tracker](#using-the-issue-tracker)
1. [Changing the code-base](#changing-the-code-base)
1. [Code Style](#code-style)

There are two primary ways to help:

> * Using the issue tracker, and

Reviewer: ```suggestion
* Using the issue tracker
```

> * Changing the code-base.

Reviewer: ```suggestion
* Changing the codebase
```

## Using the issue tracker

Use the issue tracker to suggest feature requests, report bugs, and ask questions. This is also a great way to connect with the developers of the project as well as others who are interested in this solution.

> Use the issue tracker to find ways to contribute. Find a bug or a feature, mention in the issue that you will take on that effort, then follow the "Changing the code-base" guidance below.

Reviewer: ```suggestion
Use the issue tracker to find ways to contribute. Find a bug or a feature, mention in the issue that you will take on that effort, then follow the "Changing the codebase" guidance below.
```

## Changing the code-base

Generally speaking, you should fork this repository, make changes in your own fork, and then submit a pull-request. All new code should have been thoroughly tested end-to-end in order to validate implemented features and the presence or lack of defects.

### Working with forks

> * [Configure this repository as a remote for your own fork](https://help.github.com/articles/configuring-a-remote-for-a-fork/), and

Reviewer: ```suggestion
* [Configure this repository as a remote for your own fork](https://help.github.com/articles/configuring-a-remote-for-a-fork/)
```
